topics/assembly/tutorials/ecoli_comparison/tutorial.md

```diff
@@ -0,0 +1,965 @@
+---
+layout: tutorial_hands_on
+topic_name: assembly
+tutorial_name: ecoli_comparison
+---
+
+> ### Outline of this tutorial
+>
+> In this tutorial we begin with a new genome assembly just produced in the [Unicycler tutorial]({{site.baseurl}}/topics/assembly/tutorials/unicycler-assembly/tutorial.html). This is an assembly of *E. coli* C, which we will be comparing to assemblies of all other complete genes of this species.
+>
+>--------------
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Finding closely related genomes
+
+*E. coli* is one of the most studied organisms. There are hundreds of complete genomes (in fact, the total number of *E. coli* assemblies in Genbank is over 10,500). Here we will shows how to uploaded all (!) complete *E. coli* genomes at once.
+
+## Getting complete *E. coli* genomes into Galaxy
+
+---------------------
+
+Our initial objective is to compare our assembly against all complete *E. coli* genomes to identify the most related ones and to find any interesting genome alterations. In order to do this we need to align our assembly against all other genomes. And in order to do that we need to first obtain all these other genomes.
+
+[NCBI](https://www.ncbi.nlm.nih.gov/) is the resource that would store all complete *E. coli* genomes. Specifically, they can be found [here](https://www.ncbi.nlm.nih.gov/genome/genomes/167). As we will see, this list contains over 500 genomes and so uploading them by hand will likely result in carpal tunnel syndrome, which we want to prevent. Galaxy has several features that are specifically designed for uploading and managing large sets of similar types of data. The following two **Hands-on** sections show how they can be used to import all completed *E. coli* genomes into Galaxy.
+
+> ### {% icon hands_on %} Hands-on: Preparing a list of all complete *E. coli* genomes
+>
+>Open [the NCBI list of of *E. coli* genomes](https://www.ncbi.nlm.nih.gov/genome/genomes/167) in a new window and position two browser windows (one the tutorial and the one you just opened) side by side. Then follow the steps in the following video.
```

Comment:
We should have a tool for this in Galaxy or a data_source for Genbank.

Reply:
not sure we really need this. another thing to maintain. spreadshteet works just fine

Reply:
Not for this tutorial. But imho it would be nice to have more powerful data_source tools.